packages/nimble-components/src/patterns/button/tests/doc-strings.ts

```diff
@@ -2,7 +2,7 @@ export const appearanceDescription = `This attribute affects the appearance of t
 
 <details>
     <summary>Appearance Usage</summary>
-    There is a hierarchy to which button appearance should be used.
+    <p>There is a hierarchy to which button appearance should be used.</p>
```

Comment:
When I was running `npm run storybook` locally, I saw warnings/errors in dev tools about this DOM structure:
```
Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.
    at ul
    at p
    at details
    at div
    at __WEBPACK_DEFAULT_EXPORT__ (http://localhost:6006/vendors-node_modules_storybook_blocks_dist_index_mjs.iframe.bundle.js:3340:12874)
    at div
    at http://localhost:6006/vendors-node_modules_storybook_blocks_dist_index_mjs.iframe.bundle.js:1608:2994
    at td
    at tr
    at ArgRow (http://localhost:6006/vendors-node_modules_storybook_blocks_dist_index_mjs.iframe.bundle.js:472:8207)
    at tbody
    at table
```
It looks like once it's rendered, a `<p>` gets automatically added around these lines, and the `<ul>` ends up as child of `<p>`, which is [technically invalid](https://html.spec.whatwg.org/multipage/grouping-content.html#:~:text=List%20elements%20(in%20particular%2C%20ol,it%20should%20be%20marked%20up. ):
![ButtonTextSS](https://github.com/ni/nimble/assets/20709258/ffff2512-f8ac-490a-9002-d78de01e26bc)

So I added the `<p>` around just the text and the errors went away (renders the same, the `p` and `ul` become siblings)